endpoints.go

```diff
@@ -2,18 +2,27 @@ package main
 
 import (
 	"encoding/json"
+	"io/ioutil"
 	"fmt"
 	"net/http"
 	"time"
+	"strings"
 
+    "github.com/ONSdigital/ras-rm-survey/logger"
+    "github.com/gofrs/uuid"
 	"github.com/ONSdigital/ras-rm-survey/models"
 	"github.com/gorilla/mux"
 	"github.com/spf13/viper"
 )
 
 func handleEndpoints(r *mux.Router) {
-	r.HandleFunc("/info", showInfo)
-	r.HandleFunc("/health", showHealth)
+	r.HandleFunc("/info", showInfo).Methods("GET")
+	r.HandleFunc("/health", showHealth).Methods("GET")
+	r.HandleFunc("/survey", getSurvey).Methods("GET")
+	r.HandleFunc("/survey", postSurvey).Methods("POST")
+	r.HandleFunc("/survey/{reference}", getSurveyByRef).Methods("GET")
```

Comment:
```suggestion
	r.HandleFunc("/survey/{surveyRef}", getSurveyByRef).Methods("GET")
```

It would be good to make this consistent in the code with the query parameter (this goes for both the lines below too, as well as the `surveyRef := vars["reference"]` lines in the handler)